wcc-contentful-middleman/lib/wcc/contentful/middleman/extension.rb

```diff
@@ -34,6 +34,8 @@ def initialize(app, options_hash = {}, &block)
         config.public_send("#{k}=", v) if config.respond_to?("#{k}=")
       end
 
+      # TODO: create MiddlemanRichTextRenderer and set as default here
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/watermarkchurch/wcc-contentful/blob/4b7ad73977bb23e3609efacfef4a6001e768e62a/wcc-contentful-middleman/lib/wcc/contentful/middleman/extension.rb#L37"></span>TODO added in <a href="https://github.com/watermarkchurch/wcc-contentful/blob/4b7ad73977bb23e3609efacfef4a6001e768e62a/wcc-contentful-middleman/lib/wcc/contentful/middleman/extension.rb">wcc-contentful-middleman/lib/wcc/contentful/middleman/extension.rb</a> - is there a card associated with that?</td>
    </tr>
  </tbody>
</table>